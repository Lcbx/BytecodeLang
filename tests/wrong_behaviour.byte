45 + 5 * 8 < 1000 or 34
'hello' + 23