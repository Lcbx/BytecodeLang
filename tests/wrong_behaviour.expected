compile_Out:
Error line 1, column 1 : illegal token : '"'
1 errors
__SECTION__
compile_Err:

__SECTION__
simulation_Out:

__SECTION__
simulation_Err:
Traceback (most recent call last):
  File "C:\Users\lucco\Desktop\tests\Lang\compiler\vm_simulator.py", line 13, in <module>
    with open(args.input,'rb') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests\\wrong_behaviour.hex'